### PR TITLE
remove duplicated import

### PR DESCRIPTION
**Pull request by ChiuWang**
**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/server/options/serving_test.go [hunk 1]
```diff
@@ -43,7 +43,6 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/apimachinery/pkg/version"
 	genericapirequest "k8s.io/apiserver/pkg/endpoints/request"
-	"k8s.io/apiserver/pkg/server"
```
Comment (cheftako) [inline staging/src/k8s.io/apiserver/pkg/server/options/serving_test.go]: https://github.com/golang/go/wiki/CodeReviewComments#import-dot do we actually need the dot import? If not I think we are better off eliminating line 47 and keeping 46.

Comment (charrywanganthony) [reply staging/src/k8s.io/apiserver/pkg/server/options/serving_test.go]: It's my fault. I reverted this one.

**Comment by charrywanganthony** [comment]
I use this function to find them all.
```
package main

import (
	"bufio"
	"flag"
	"fmt"
	"os"
	"path/filepath"
	"strings"
)

func main() {
	flag.Parse()
	root := flag.Arg(0)
	getFilelist(root)
}

func getFilelist(rootPath string) {
	var filePath []string
	err := filepath.Walk(rootPath, func(path string, f os.FileInfo, err error) error {
		if f == nil {
			return err
		}
		if f.IsDir() {
			return nil
		}
		if strings.HasSuffix(path, ".go") {
			filePath = append(filePath, path)
		}
		return nil
	})
	if err != nil {
		fmt.Printf("filepath.Walk() returned %v\n", err)
	}
	fmt.Printf("filePath.Walk() runs well.\n")

	for _, path := range filePath {
		if err := dealFile(path); err!=nil {
			fmt.Printf("dealFile returned %v\n", err)
		}
	}

}

func dealFile(path string) error {
	file, err := os.Open(path)
        if err != nil {
                return err
        }
        defer file.Close()

	scanner := bufio.NewScanner(file)
	lines := make([]string, 0)
	var started bool
	for scanner.Scan() {
		text := scanner.Text()
		if started {
			if text == `)` {
				break
			} else {
				lines = append(lines, text)
				continue
			}
		}
		if text == `import (` {
			started = true
		} else if strings.HasPrefix(text, `import `) && !started {
			lines = append(lines, text)
		}
	}

	deal(lines, path)

	return scanner.Err()
}

func deal(lines []string, name string) {
	m := make(map[string]bool)
	for _, l := range lines {
		ll := strings.Split(l, " ")
		for _, l := range ll {
			l = strings.TrimSpace(l)
			if strings.HasPrefix(l, `"`) {
				if m[l] == true {
					fmt.Print(name, "\t", l, "\n")
				} else {
					m[l] = true
				}
			}
		}
	}
}
```

**Review by cheftako — APPROVED** [review]
/lgtm

File: pkg/apis/storage/fuzzer/fuzzer.go [hunk 2]
```diff
@@ -21,14 +21,13 @@ import (
...
 	api "k8s.io/kubernetes/pkg/apis/core"
-	"k8s.io/kubernetes/pkg/apis/storage"
 	storageapi "k8s.io/kubernetes/pkg/apis/storage"
 )
 
 // Funcs returns the fuzzer functions for the storage api group.
 var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 	return []interface{}{
-		func(obj *storage.StorageClass, c fuzz.Continue) {
+		func(obj *storageapi.StorageClass, c fuzz.Continue) {
```
Comment (sttts) [inline pkg/apis/storage/fuzzer/fuzzer.go]: Don't we use the storageinternal style elsewhere?

File: pkg/kubelet/config/common_test.go [hunk 3]
```diff
@@ -227,7 +226,7 @@ func TestStaticPodNameGenerate(t *testing.T) {
 	}
 	for _, c := range testCases {
 		assert.Equal(t, c.expected, generatePodName(c.podName, c.nodeName), "wrong pod name generated")
-		pod := &core.Pod{}
+		pod := &api.Pod{}
```
Comment (sttts) [inline pkg/kubelet/config/common_test.go]: core or coreinternal. `api` is legacy.

File: test/e2e/apps/daemon_set.go [hunk 4]
```diff
@@ -738,7 +737,7 @@ func curHistory(historyList *apps.ControllerRevisionList, ds *apps.DaemonSet) *a
 func waitFailedDaemonPodDeleted(c clientset.Interface, pod *v1.Pod) func() (bool, error) {
 	return func() (bool, error) {
 		if _, err := c.CoreV1().Pods(pod.Namespace).Get(pod.Name, metav1.GetOptions{}); err != nil {
-			if errors.IsNotFound(err) {
+			if apierrs.IsNotFound(err) {
```
Comment (sttts) [inline test/e2e/apps/daemon_set.go]: apierrors elsewhere

File: test/e2e/storage/vsphere/vsphere_utils.go [hunk 5]
```diff
@@ -41,10 +41,10 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/storage/utils"
 
-	"github.com/vmware/govmomi/find"
-	vimtypes "github.com/vmware/govmomi/vim25/types"
 	"regexp"
 	"strings"
+
+	"github.com/vmware/govmomi/find"
```
Comment (sttts) [inline test/e2e/storage/vsphere/vsphere_utils.go]: strange order.

**Comment by charrywanganthony** [comment]
@sttts  Modified.

**Comment by charrywanganthony** [comment]
Rebased.
cc: @php-coder @sttts @cheftako @derekwaynecarr

**Comment by charrywanganthony** [comment]
Rebased. would you approve this?
cc: @php-coder @sttts @cheftako @derekwaynecarr

**Review by cheftako — APPROVED** [review]
/lgtm

**Comment by charrywanganthony** [comment]
Rebased. need aprove again.
@sttts @ash2k @cheftako :smiley_cat:

**Comment by sttts** [comment]
/assign @tallclair 
/assign @liggitt 

for approval.

**Comment by tallclair** [comment]
/approve

We should add that function to our presubmit.

**Pull request merged**